earth_enterprise/src/fusion/autoingest/AssetGenDaemonH.pl

```diff
@@ -283,6 +275,20 @@ ()
 
 print $fh <<EOF;
 
+// Convenience class that ties together related types
```

Comment:
This struct puts all the classes that relate to a specific asset type into one place where it's easy to reference several of them at once. It makes the Asset Factory functions much cleaner, but it requires a lot of changes to use it.

Reply:
This makes the template parameters a LOT cleaner.